.github/workflows/pr-windows.yml

```diff
@@ -95,11 +95,20 @@ jobs:
       # See https://cmake.org/cmake/help/latest/manual/ctest.1.html for more detail
       run: ctest -C ${{env.BUILD_TYPE}} --verbose # --output-on-failure
 
-    - uses: actions/upload-artifact@v2
-      with:
-        name: coverage
-        path: ${{github.workspace}}/build/__coverage
```

Comment:
Decided to move this to the bottom (instead of just allowing to fail) so we can get dist if this fails in the future for whatever reason